### DISCUSSION TITLE
Filter domains masking as medium.com

### DISCUSSION THREAD

**Discussion by bdmorin** [dailydotdev/daily#990] [Feature Requests]
Medium.com is a blight to content. I never want to see it, and block it where ever I can. When domains are masked as medium.com it's even worse. Paywalls, and privacy issues riddle medium. 

It would be great to be able to filter/block/do not show posts that are domains masking medium.com

**Comment by idoshamun** [comment]
Hi @bdmorin,

It's an interesting suggestion that we currently don't support.
I'll take it internally

**Reply by tomByrer** [reply]
Any progress?
Those Medium popups drive me crazy.

**Reply by idoshamun** [reply]
Unfortunately, not yet 😢

**Reply by tomByrer** [reply]
I'm already over my limit for free Medium articles.  I find only ~15% of Medium articles I *need* to read.

**Comment by tomByrer** [comment]
Would also be helpful to avoid tracking sites like this:
![image](https://github.com/dailydotdev/daily/assets/1308419/79690d8c-23f6-447e-8d61-dbdaf67daf73)

**Reply by idoshamun** [reply]
You probably clicked on an ad. It's marked as "promoted" on the site. Can you confirm please?

**Comment by marvinengelmann** [comment]
Any progress on domain filter? I also don't want Medium posts.

**Reply by idoshamun** [reply]
Not yet but we're experimenting with showing them as a first step
<img width="342" alt="image" src="https://github.com/user-attachments/assets/535111d7-1235-4487-b729-20bba84a660c">

**Reply by ocomsoft** [reply]
@idoshamun  love the idea but I would also like to block this site. I was about to submit a request and thought I should search and   I found this issue.

**Reply by idoshamun** [reply]
@ocomsoft makes total sense
